### PR TITLE
dash/non-nested-element

### PR DESCRIPTION
**Pull request by karolkolodziej**
Added `isStandalone` option to add non-nested options in the edit mode sidebar, #20793.

### CODE REVIEW

**Review by vazonik — COMMENTED** [review]
Technically it looks & works ok! But I have two suggestions/ideas.
1. I think we should write something about it in the EditMode docs article and maybe create a standalone sample?
2. Maybe it would be good to apply this to some default options?

**Review by jomar-honsi — APPROVED** [review]

**Review by jomar-honsi — APPROVED** [review]

**Review by vazonik — APPROVED** [review]

**Comment by stitot** [comment]
Hm..... We are adding some new concepts/wording here (nested, standalone) in addition to new option. I feel it's perhaps to verbose. Not possible with a simpler approach? E.g.

```
editableOptions: [{
        name: 'Component title',
        propertyPath: ['title'],
        type: 'input'
      }, {
        name: 'chartOptions',
        children: [{
          name: 'Marker Radius',
          options: [{
            name: 'Marker Radius',
            propertyPath: [
              'chartOptions',
              'plotOptions',
              'series',
              'marker',
              'radius'
            ],
            type: 'select',
            selectOptions: [{
              name: 3
            }, {
              name: 5
            }]
          }]
        }]
      }
    ]
```
If the `children` option doesn't exist it's standalone (the default). Isn't children a more common terminology for this concept, from a dev point of view? If feel standalone/nested calls for more explanation.

**Comment by karolkolodziej** [comment]
@stitot I'm fine with that but the only drawback is that with this approach it won't be possible to have collapsible singular elements like we used to have: 
image

**Comment by stitot** [comment]
Hm...... From a UX (and perhaps a11y?) perspective it makes sense for me to have all these single values exposed. Takes more space, but easier to find the value you try to edit (by looking at actual value, not name of field) instead of expanding them to see what's hidden inside.

We could introduce an `collapsed` option, that is false by default?

File: css/dashboards/dashboards.css [hunk 1]
```diff
@@ -1214,7 +1214,8 @@
     flex-direction: column;
 }
 
-.highcharts-dashboards-edit-accordion-content > .highcharts-dashboards-edit-label-text {
+/* stylelint-disable-next-line max-line-length */
+.highcharts-dashboards-edit-accordion-content:not(.highcharts-dashboards-edit-standalone-element) > .highcharts-dashboards-edit-label-text {
```
Comment (sebastianbochan) [inline css/dashboards/dashboards.css:1218]: So beautiful <3

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**